### PR TITLE
Add the localization framework doc

### LINKED ISSUE #4839: Write localization framework doc
This doc should explain how the whole translation workflow works.

It should:
- explain the idea of the translations being crowdsourced
- explain how we'll use a project(s) for the translations
- mention the translation guide from #4834
- how we organize our `Resource.<language-code>.resx` files (and comments inside)
- explain when and how a translation PR can be merged back into develop

### PR DESCRIPTION
**Pull request by daividssilverio**
## What's this?
This adds the docs with details on how the localization framework will/should work.

### Relationships
Closes #4839

## Why do we want this?
To keep the translation process somehow organized.

## How is it done?
Check the docs.

### Why not another way?
Please, read it through and make suggestions, say if you don't like something, improve it somehow.

## :squid: Permissions
🦑 but after it's done, and @maritoggl and @amulware have approved (or self dismissed from reviewing it)

### CODE REVIEW

**Comment by daividssilverio** [comment]
I'm assuming this will be merged after #5681 and #5682 (but it doesn't matter either way).

File: .github/ISSUE_TEMPLATE/g-translation-request.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+name: "🌏 Translation Request"
+about: Do you want the Toggl apps to be translated into another language? That's the right place!
+
+---
+
 ## Target Language: <!-- Requested Language Here -->
 <!-- Please check that no other request for the same language has been made before submitting this issue. Leave a comment in the issue if you'd like collaborate -->
```
Comment (amulware) [inline .github/ISSUE_TEMPLATE/g-translation-request.md]: can't comment on the below, but what's up with the 'requested at' header? Pretty sure GitHub gives us that info, right?

Comment (amulware) [reply .github/ISSUE_TEMPLATE/g-translation-request.md]: I'm also not sure about the template below - that seems to imply that every translation will be a single PR. Do we not support a more flexible workflow? Translating the whole app is a lot of effort.

File: docs/internationalization/localization-framework.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+We use `RESX` files, the built-in mechanism for localizing .NET applications.
+If you are curious, you might want to look at the [Microsoft docs](https://docs.microsoft.com/en-us/dotnet/framework/resources/creating-resource-files-for-desktop-apps#resources-in-resx-files).
+
+### Where can I find current translations?
+The text entries resources files can be found in the `Toggl.Shared` folder.
+The base file is `Toggl.Shared/Resources.resx`.
+Each translation will have its own file which follows the following format: `Toggl.Shared/Resources.language-code.resx`. Ex: Japanese translations should go into `Toggl.Shared/Resources.ja.resx`.
+
+### How will we organize translations
+We'll be utilizing a github project to organize the work related to translations. The project will have columns for each phase of the translation process.
```
Comment (amulware) [inline docs/internationalization/localization-framework.md]: Future tense? Will you change this once we have that project? I'd just write it in present tense right away.

File: docs/internationalization/localization-framework.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+- New Translations:
+When the whole app is to be translated into a new language, a new [New Translation Request issue](https://github.com/toggl/mobileapp/issues/new?template=g-translation-request.md) should be created.
+
+- Improvements:
+When only one or few text entries of the app in a specific language need to be corrected/improved, a new [New Translation Fix/Improvement issue](https://github.com/toggl/mobileapp/issues/new?template=h-translation-fix-request.md) should be created.
+
+- New Copy:
+When introducing or updating new text entries in the app, we'll need to get all of them translated/updates to languages currently supported before a new release. This shouldn't be a lot of work in general, except when we introduce completely new screens/features.
+Ideally, new copies translations/updates should be done before we even start development that will use the new text.
```
Comment (amulware) [inline docs/internationalization/localization-framework.md]: How can this be achieved?

File: docs/internationalization/localization-framework.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+Each translation will have its own file which follows the following format: `Toggl.Shared/Resources.language-code.resx`. Ex: Japanese translations should go into `Toggl.Shared/Resources.ja.resx`.
+
+### How will we organize translations
+We'll be utilizing a github project to organize the work related to translations. The project will have columns for each phase of the translation process.
+
+#### The Translations Github Project
+The project will have the following columns:
+
+- New Translations:
+When the whole app is to be translated into a new language, a new [New Translation Request issue](https://github.com/toggl/mobileapp/issues/new?template=g-translation-request.md) should be created.
```
Comment (amulware) [inline docs/internationalization/localization-framework.md]: See my comment above on a full translation being a big step that might prevent users from participating.

File: docs/internationalization/localization-framework.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+- In progress:
+Translation/update/fix issue has been assigned to some translator and is currently being worked on.
+
+- Paused:
+Translation/update/fix issues that have been started, have a branch with some progress in it, but the original translator can't continue. Another translators might continue with them.
+
+- Translation Pull Requests:
+All translation issues related PRs have their own PR templates and should be included in this column. Just to keep things organized.
+Please use the following templates to create PR templates:
```
Comment (amulware) [inline docs/internationalization/localization-framework.md]: And how does one use the template?

File: docs/internationalization/localization-framework.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  - [PR template for new translations](../../.github/PULL_REQUEST_TEMPLATE/translation-pull-request-template.md)
+  - [PR template for fixes/improvements](../../.github/PULL_REQUEST_TEMPLATE/translation-fix-pull-request-template.md)
+  - [PR template for new copies](../../.github/PULL_REQUEST_TEMPLATE/new-copy-translation-pull-request-template.md)
+
+- In Review:
+As the name say, all translations PR that are currently in review should live here.
```
Comment (amulware) [inline docs/internationalization/localization-framework.md]:
```suggestion
As the name says, all translations PR that are currently in review should live here.
```

File: docs/internationalization/localization-framework.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+- Sufficiently Approved:
+When a PR from `In Review` has been sufficiently approved (A dev and ideally a translator should approve a translation PR). Don't merge this PR until it has been approved by the team lead or responsible mobile product team member.
+
+- Merged (Waiting to be released):
+The translation/update/fix PR has been merged into develop and should be released in coming releases.
+Ideally, update the issue and PR saying when users should expect to see it in the wild.
+
+- Done / Archived:
+Move all issues/PRs that have been closed here.
```
Comment (amulware) [inline docs/internationalization/localization-framework.md]: When are issues closed?

File: docs/internationalization/localization-framework.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
+## Mobile Translation Framework
+
+### Who is translating the app?
+The idea is that the app will be translated through crowdsourcing. That is, both users and translation volunteers will be contributing to our codebase. Although that's the idea, nothing stops us from using means other than crowdsourcing as well, the framework should be flexible enough for that.
```
Comment (maritoggl) [inline docs/internationalization/localization-framework.md]: I find the last sentence unnecessary and would simplify:
Toggl Mobile apps are translated through crowdsourcing. That means, both our users and translation volunteers are welcome to contribute to our codebase.

File: docs/internationalization/localization-framework.md (hunk 4 above)
Comment (maritoggl) [reply docs/internationalization/localization-framework.md]: "If you would like the whole app to be translated into a new language, create a New Language Request issue (include link to template). If you see that the Translations project (include direct link) already has a request for the language you are rooting for, don't hesitate to comment or react to that issue."

File: docs/internationalization/localization-framework.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+We'll be utilizing a github project to organize the work related to translations. The project will have columns for each phase of the translation process.
+
+#### The Translations Github Project
+The project will have the following columns:
+
+- New Translations:
+When the whole app is to be translated into a new language, a new [New Translation Request issue](https://github.com/toggl/mobileapp/issues/new?template=g-translation-request.md) should be created.
+
+- Improvements:
+When only one or few text entries of the app in a specific language need to be corrected/improved, a new [New Translation Fix/Improvement issue](https://github.com/toggl/mobileapp/issues/new?template=h-translation-fix-request.md) should be created.
```
Comment (maritoggl) [inline docs/internationalization/localization-framework.md]: If there are any erroneous texts that need to be improved in a specific language, create a new "Translation Improvement Request" issue (direct link to template).

File: .github/ISSUE_TEMPLATE/g-translation-request.md [hunk 10]
```diff
@@ -1,3 +1,9 @@
+---
+name: "🌏 Translation Request"
```
Comment (maritoggl) [inline .github/ISSUE_TEMPLATE/g-translation-request.md]: I'd rename "Translation Request" to "New Language Request"

File: .github/ISSUE_TEMPLATE/g-translation-request.md (hunk 1 above)
Comment (maritoggl) [reply .github/ISSUE_TEMPLATE/g-translation-request.md]: Couldn't comment on the below either, but please remove the requested at row - we can rely on the issue creation date. 

Progress section - I would remove it. Progress is reflected by issue statuses coming from the project board. I'd propose following statuses for the board: 
- Preparing translation(s)
- Pending review
- Waiting for release
- Released

☝ should work both for new language and improvement requests. The issue would move to "Released" status only after both (if relevant for the specific translation) platforms have been released.

Reason for the above proposal: a typical user has no idea what are pull requests, what does merging into develop etc mean. They don't care about that either.

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md [hunk 11]
```diff
@@ -1,3 +1,9 @@
+---
+name: "🌏 Translation Fix/Improvement Request"
```
Comment (maritoggl) [inline .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: I'd change the title to simply "Translation Improvement Request"

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md [hunk 12]
```diff
@@ -1,3 +1,9 @@
...
+about: Oh, found something odd in some translation in the Toggl apps? Please help us creating an issue to fix it :)
```
Comment (maritoggl) [inline .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: I'd change the about to "Oh, noticed an odd translation in our app? Please help us by creating an issue to fix it :)"

Comment (maritoggl) [reply .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: Again, can't comment on below, but:

Same as for the new language request - remove the "Requested at" row (we don't need it).

Proposal for the rest: 

Tell us more about the problem:

- What seems to be the issue?
_please inform the screen(s) where the translation looks odd/wrong and explain the problem_
_if possible, provide us with screenshots for context_
- Problematic text
- Better translation

The comments between <!-- don't seem to show in GitHub btw.

File: docs/internationalization/localization-framework.md [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+
+- New Copy:
```
Comment (maritoggl) [inline docs/internationalization/localization-framework.md]: I'd make this plural as well "New Copies"

File: docs/internationalization/localization-framework.md [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+When introducing or updating new text entries in the app, we'll need to get all of them translated/updates to languages currently supported before a new release. This shouldn't be a lot of work in general, except when we introduce completely new screens/features.
```
Comment (maritoggl) [inline docs/internationalization/localization-framework.md]: When introducing new text entries in the app, all of them need to be translated to currently supported languages before the new texts can be released. <- I'd keep it like that, comment about whether it's a lot of work or not seems irrelevant in this doc.

File: docs/internationalization/localization-framework.md (hunk 3 above)
Comment (maritoggl) [reply docs/internationalization/localization-framework.md]: I agree with that, but I feel this comment doesn't belong in this translation guide, but instead to some project or feature development guide or smth like that.

File: docs/internationalization/localization-framework.md [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+Ideally, new copies translations/updates should be done before we even start development that will use the new text.
+When a new copy is introduced, a [New Copy Translation issue](https://github.com/toggl/mobileapp/issues/new?template=i-new-copy-translation-request.md) should be created. Issues of this kind should only be closed when all translations/updates are done.
+
+- In progress:
+Translation/update/fix issue has been assigned to some translator and is currently being worked on.
+
+- Paused:
+Translation/update/fix issues that have been started, have a branch with some progress in it, but the original translator can't continue. Another translators might continue with them.
```
Comment (maritoggl) [inline docs/internationalization/localization-framework.md]: This sounds confusing to me. I'd keep this in the proposed "Preparing translation(s)" status and manage it with comments if anything is incomplete.

File: docs/internationalization/localization-framework.md [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+
+- Improvements:
+When only one or few text entries of the app in a specific language need to be corrected/improved, a new [New Translation Fix/Improvement issue](https://github.com/toggl/mobileapp/issues/new?template=h-translation-fix-request.md) should be created.
+
+- New Copy:
+When introducing or updating new text entries in the app, we'll need to get all of them translated/updates to languages currently supported before a new release. This shouldn't be a lot of work in general, except when we introduce completely new screens/features.
+Ideally, new copies translations/updates should be done before we even start development that will use the new text.
+When a new copy is introduced, a [New Copy Translation issue](https://github.com/toggl/mobileapp/issues/new?template=i-new-copy-translation-request.md) should be created. Issues of this kind should only be closed when all translations/updates are done.
+
+- In progress:
```
Comment (maritoggl) [inline docs/internationalization/localization-framework.md]: Would use "Preparing translation(s)" status to reflect that.

File: docs/internationalization/localization-framework.md (hunk 7 above)
Comment (maritoggl) [reply docs/internationalization/localization-framework.md]: My proposal for the statuses is somewhere above, but sharing again:
- Preparing translation(s)
- Pending review
- Waiting for release
- Released
Reason for the above proposal: typical user has no idea what are pull requests, what does merging into develop etc mean. They don't care either.

**Comment by maritoggl** [comment]
Disclaimer: when reviewing these files, my assumption was that these guidelines should be simple for primarily external and (mostly) non-developer contributors and all comments and proposals have been added with this in mind.

File: .github/ISSUE_TEMPLATE/g-translation-request.md (hunk 1 above)
Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/g-translation-request.md]: > Can't comment on the below, but what's up with the 'requested at' header? Pretty sure GitHub gives us that info, right?

It does, I thought a little bit of redundancy would be nice for this. Github only shows this information when you open the issue details; It doesn't appear in the issue preview in the project page for example.

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md (hunk 12 above)
Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: > Same as for the new language request - remove the "Requested at" row (we don't need it).

Alrighty.

Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: > The comments between <!-- don't seem to show in GitHub btw.

@maritoggl , that was the idea. You rather make that explicit and fixed in the Issue template?

File: docs/internationalization/localization-framework.md (hunk 3 above)
Comment (daividssilverio) [reply docs/internationalization/localization-framework.md]: If we had designs ready weeks before development; We could simply update the `Resources.resx` base file in develop and start a `New Copies` issue. But Yeah... maybe this doesn't belong here (yet).

File: docs/internationalization/localization-framework.md (hunk 7 above)
Comment (daividssilverio) [reply docs/internationalization/localization-framework.md]: > When are issues closed? 

When respective PRs (There could be multiple PRs) are merged.

**Comment by daividssilverio** [comment]
@maritoggl @amulware I've updated the docs with your suggestions.
Could you please take another look?

File: .github/ISSUE_TEMPLATE/g-translation-request.md (hunk 1 above)
Comment (amulware) [reply .github/ISSUE_TEMPLATE/g-translation-request.md]: Doesn't seem that important information for me.

File: docs/internationalization/localization-framework.md (hunk 7 above)
Comment (amulware) [reply docs/internationalization/localization-framework.md]: @maritoggl Who takes care to check when translations are released and moves it from waiting-for-release to released? We don't make this distinction anywhere in the repo right now, and this is potentially significant manual effort, and something easy to forget in which case the information will be out of date and possibly confusing.

**Comment by amulware** [comment]
How come the above discussions weren't closed automatically when you made relevant changes? 🤔

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md (hunk 12 above)
Comment (maritoggl) [reply .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: It probably wouldn't hurt, yes. This goes for other comments as well - there's a reason you added them to the file in the first place, I don't see a reason we should hide them from the template :)

File: docs/internationalization/localization-framework.md (hunk 7 above)
Comment (maritoggl) [reply docs/internationalization/localization-framework.md]: @amulware How is that currently done with other projects where issues get technically done and then released?

File: docs/internationalization/localization-framework.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+  When introducing new text entries in the app, all of them need to be translated to currently supported languages before the new texts can be released. Create a [New Copy Translation issue](https://github.com/toggl/mobileapp/issues/new?template=i-new-copy-translation-request.md) to start the translation process for new copies. 
+
+- Preparing translation(s):
+Translation/update/fix issue either has been assigned to some translator or is currently being worked on in some other way.
+
+- Pending review:
+As the name says, all translations PRs that are currently in review should live here.
+
+- Waiting for release:
+When a PR from `Pending review` has been sufficiently approved (A dev and ideally a translator should approve a translation PR), move it here, along with the original issue.
```
[the remark this answers is not in the document]
Comment (amulware) [reply docs/internationalization/localization-framework.md]: Same with the line just below, only developers can merge PRs.

File: docs/internationalization/localization-framework.md [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+Don't merge this PR into develop until it has been approved by the team lead or responsible mobile product team member.
+After the translation/update/fix PR has been merged into develop, it should go out to production in one of the coming releases.
+Ideally, update the issue and PR saying when users should expect to see it in the wild.
```
Comment (amulware) [inline docs/internationalization/localization-framework.md]: Who does this, based on what? We generally don't promise when anything gets released publicly because there are always unknowns. What makes this different?

Comment (amulware) [reply docs/internationalization/localization-framework.md]: Also update both?

File: docs/internationalization/localization-framework.md [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+
+- Released:
+Yeah! Translations here have been released!
+Move issues and related PRs here when the translation has been released.
```
Comment (amulware) [inline docs/internationalization/localization-framework.md]: - Who moves? Both issues and PRs here now?

File: docs/internationalization/localization-framework.md (hunk 7 above)
Comment (amulware) [reply docs/internationalization/localization-framework.md]: @maritoggl It isn't. An issue is closed when the PR is merged into the project or main branch. We can look up if a particular change has already been released, but we don't proactively do s.

File: docs/internationalization/localization-framework.md [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+
+- Preparing translation(s):
+Translation/update/fix issue either has been assigned to some translator or is currently being worked on in some other way.
+
+- Pending review:
+As the name says, all translations PRs that are currently in review should live here.
+
+- Waiting for release:
+When a PR from `Pending review` has been sufficiently approved (A dev and ideally a translator should approve a translation PR), move it here, along with the original issue.
+Don't merge this PR into develop until it has been approved by the team lead or responsible mobile product team member.
```
Comment (maritoggl) [inline docs/internationalization/localization-framework.md]: This again sounds like an internally directed guideline and doesn't leave an impression that a random user could help us with the translation process.

File: .github/ISSUE_TEMPLATE/g-translation-request.md (hunk 1 above)
Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/g-translation-request.md]: Removed already.

File: docs/internationalization/localization-framework.md [hunk 21]
```diff
@@ -0,0 +1,68 @@
...
+We are utilizing a [GitHub project](https://github.com/toggl/mobileapp/projects/74) to organize the translation-related work. 
+The project has columns for each phase of the translation process.
+
+#### The Translations GitHub Project
+The project will have the following columns:
+
+- New Requests:
+  
+  Translation requests are located here. Different types of translations requests are created using different issue templates:
+  - New Translations:
```
[the remark this answers is not in the document]
Comment (daividssilverio) [reply docs/internationalization/localization-framework.md]: Actually, I've changed this section to convey the idea that there would be only one section "New Requests". And the issues would tell what kind of request they are. I've also updated the issue templates to include the request type.

File: docs/internationalization/localization-framework.md [hunk 22]
```diff
@@ -0,0 +1,68 @@
...
+  If there are any erroneous texts that need to be improved in a specific language, create a new [Translation Improvement issue](https://github.com/toggl/mobileapp/issues/new?template=h-translation-fix-request.md).
+
+  - New Copies:
+  When introducing new text entries in the app, all of them need to be translated to currently supported languages before the new texts can be released. Create a [New Copy Translation issue](https://github.com/toggl/mobileapp/issues/new?template=i-new-copy-translation-request.md) to start the translation process for new copies. 
+
+- Preparing translation(s):
+Translation/update/fix issue either has been assigned to some translator or is currently being worked on in some other way.
+
+- Pending review:
+As the name says, all translations PRs that are currently in review should live here.
```
[the remark this answers is not in the document]
Comment (daividssilverio) [reply docs/internationalization/localization-framework.md]: I thought only PRs would go here actually, issues would stay in "Preparing translations" and go directly to "Waiting for release" when everything is done.

File: docs/internationalization/localization-framework.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (daividssilverio) [reply docs/internationalization/localization-framework.md]: That makes sense, I'll rephrase that telling one of our developers will move it.

File: docs/internationalization/localization-framework.md (hunk 20 above)
Comment (daividssilverio) [reply docs/internationalization/localization-framework.md]: I'll just remove this one then.

File: docs/internationalization/localization-framework.md (hunk 18 above)
Comment (daividssilverio) [reply docs/internationalization/localization-framework.md]: Yep, I'll include the information that a dev will do it. And remove the uncertainties.

File: docs/internationalization/localization-framework.md (hunk 19 above)
Comment (daividssilverio) [reply docs/internationalization/localization-framework.md]: Yeah, I didn't think of a better place to leave the issues and PRs after everything is done.
A dev will move, I'll update that.

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md [hunk 23]
```diff
@@ -1,21 +1,26 @@
...
-<!-- provide us with screenshots if possible -->
+- What seems to be the issue?
+  
+  *please inform the screen(s) where the translation looks odd/wrong and explain the problem. If possible, provide us with screenshots for context.*
 
-### How should it be?
-<!-- If the fix is easy enough, you can just type how it should be, and where. One of our devs can create the fix PR  -->
+- Problematic text
+  
+  *please paste the problematic test here, identifying it in the source code if you can.*
```
[the remark this answers is not in the document]
Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: Removing the *italics* for now.

File: .github/ISSUE_TEMPLATE/g-translation-request.md [hunk 24]
```diff
@@ -1,13 +1,12 @@
...
+# New Language Request
 
-- [ ] It has started
-- [ ] Pull request created and awaiting review (Reference PR to this issue). [PR template can be copied from here.](../PULL_REQUEST_TEMPLATE/translation-pull-request-template.md)
-- [ ] Pull request approved
-- [ ] Pull request merged into develop
-- [ ] Released on iOS in version `add release version here`
-- [ ] Released on Android in version `add release version here`
+## Target Language <!-- Requested Language Here -->
+Please check that no other request for the same language has been made before submitting this issue.
```
Comment (amulware) [inline .github/ISSUE_TEMPLATE/g-translation-request.md]: Shouldn't this line be in a comment as well?

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md [hunk 25]
```diff
@@ -1,21 +1,26 @@
...
+---
 
-## Tell us more about the problem
-### What is wrong?
-<!-- please inform the screen(s) where the translation looks odd/wrong -->
-<!-- provide us with screenshots if possible -->
+# Translation Improvement Request
+
+## Target Language <!-- Requested Language Here -->
+Please check that no other request for the same language fix has been made before submitting this issue.
```
Comment (amulware) [inline .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: ditto

File: docs/internationalization/localization-framework.md [hunk 26]
```diff
@@ -0,0 +1,61 @@
+# Mobile Translation Framework (work in progress)
```
Comment (amulware) [inline docs/internationalization/localization-framework.md]: Please check headlines across this file, this one is clearly unintended.

File: .github/ISSUE_TEMPLATE/g-translation-request.md (hunk 24 above)
Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/g-translation-request.md]: No. @maritoggl suggested to leave stuff in the open here: https://github.com/toggl/mobileapp/pull/5683#discussion_r311456015

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md (hunk 25 above)
Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: Same thing: https://github.com/toggl/mobileapp/pull/5683#discussion_r311456015

File: docs/internationalization/localization-framework.md (hunk 26 above)
Comment (daividssilverio) [reply docs/internationalization/localization-framework.md]: 🙈

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md [hunk 27]
```diff
@@ -1,21 +1,26 @@
...
 
-### How should it be?
-<!-- If the fix is easy enough, you can just type how it should be, and where. One of our devs can create the fix PR  -->
+Leave a comment in the issue if you'd like collaborate.
```
Comment (maritoggl) [inline .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: Let's move that to the end of the template as that really is a call for action to other users, not for the person submitting the issue.

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md [hunk 28]
```diff
@@ -1,21 +1,26 @@
...
 
-- [ ] Translation fix has been analyzed and approved
-- [ ] Pull request created and awaiting review (Reference PR to this issue). [PR template can be copied from here.](../PULL_REQUEST_TEMPLATE/translation-fix-pull-request-template.md)
-- [ ] Pull request approved
-- [ ] Pull request merged into develop
-- [ ] Released on iOS in version `add release version here`
-- [ ] Released on Android in version `add release version here`
+- Problematic text
+```  
+  Please paste the problematic text here.
```
Comment (maritoggl) [inline .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: Reword a bit: "Type/paste the problematic text here"

Issues are created using computers mostly, can't paste from mobile (unless pasted from codebase)

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md [hunk 29]
```diff
@@ -1,21 +1,26 @@
...
 
-### How should it be?
-<!-- If the fix is easy enough, you can just type how it should be, and where. One of our devs can create the fix PR  -->
+Leave a comment in the issue if you'd like collaborate.
 
-## Progress
+## Tell us more about the problem
+- What seems to be the issue?
+  
+  Please let us know exactly where the translation looks odd/wrong and explain the problem. If possible, provide us with screenshots for context.
```
Comment (maritoggl) [inline .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: That can be inside the HTML tags I guess.

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md [hunk 30]
```diff
@@ -1,21 +1,26 @@
...
-- [ ] Pull request merged into develop
-- [ ] Released on iOS in version `add release version here`
-- [ ] Released on Android in version `add release version here`
+- Problematic text
+```  
+  Please paste the problematic text here.
+```
+- Better translation
+```  
+  Please paste the better translation here.
```
Comment (maritoggl) [inline .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: type/paste

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md (hunk 29 above)
Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: which HTML tags? do you mean these `<!-- -->` ?

Comment (amulware) [reply .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: That's the one she means - I believe that Mari didn't know html comments would be visible when editing the issue when she spoke out against them before. (And I agree, that if they were not we should remove them entirely.) But they ARE visible, so we should use them.

File: .github/ISSUE_TEMPLATE/h-translation-fix-request.md [hunk 31]
```diff
@@ -1,21 +1,26 @@
...
-<!-- provide us with screenshots if possible -->
+# Translation Improvement Request
+
+## Target Language 
+<!-- Requested Language Here -->
 
-### How should it be?
-<!-- If the fix is easy enough, you can just type how it should be, and where. One of our devs can create the fix PR  -->
+## Tell us more about the problem
+- What seems to be the issue?
```
Comment (amulware) [inline .github/ISSUE_TEMPLATE/h-translation-fix-request.md]: I'd just remove this line. The header above says the same thing, and this sounds like we're not taking the user seriously.
